### PR TITLE
Be compatible with python 2

### PR DESCRIPTION
**Pull request by Xuanwo**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -21,4 +21,5 @@
         'PyYAML >= 3.1',
         'qingstor-sdk',
         'docutils >= 0.10',
+        'tqdm',
```
Comment (prettyxw) [inline setup.py]: Please specify a version of the 3rd-party package (`tqdm` and `qingstor-sdk`).

File: qingstor/qsctl/utils.py [hunk 2]
```diff
@@ -84,16 +95,23 @@ def confirm_by_user(notice):
...
-        if sys.version < "3":
-            key_name = encode_to_gbk(key_name)
+        if is_python2:
+            try:
+                key_name = encode_to_gbk(key_name)
+            except UnicodeDecodeError:
+                print(
+                    "Python2 on Windows do not support this key_name: %s",
+                    key_name
+                )
```
Comment (prettyxw) [inline qingstor/qsctl/utils.py]:
``` python
print(
    "WARN: Python 2 on Windows does not support this object key: %s",
    key_name
)
```

File: scenarios/requirements.txt [hunk 3]
```diff
@@ -0,0 +1,5 @@
+qingstor-sdk
```
Comment (prettyxw) [inline scenarios/requirements.txt]: Please specify a version.

File: scenarios/steps/cp.py [hunk 4]
```diff
@@ -0,0 +1,155 @@
...
+    sh.mkdir("tmp")
+    sh.dd("if=/dev/zero", "of=tmp/中文测试", "bs=1048576", "count=1")
+    sh.dd("if=/dev/zero", "of=tmp/qsctl_test_big", "bs=1048576", "count=10")
+
+
+@when(u'copy to QingStor key')
+def step_impl(context):
+    command = qsctl(
+        "cp",
+        "tmp/{filename}".format(filename="中文测试"),
```
Comment (prettyxw) [inline scenarios/steps/cp.py]: Test data and expected behavior should be stored in feature files, such as object keys, file size, success status and error code.

File: scenarios/steps/cp.py [hunk 5]
```diff
@@ -0,0 +1,133 @@
...
+    for row in context.table:
+        command = qsctl(
+            "cp",
+            "tmp/{filename}".format(filename=row["name"]),
+            "qs://{bucket}/{filename}".format(
+                bucket=test_data['bucket_name'], filename=row["name"]
+            )
+        )
+        command.wait()
+
```
Comment (frostyplanet) [inline scenarios/steps/cp.py]: 一般来说是空1行， flake8可能会报warning

Comment (Xuanwo) [inline scenarios/steps/cp.py]: It's a mistake, I'll fix it.

**Pull request merged**